config/webpacker.yml

```diff
@@ -54,9 +54,6 @@ development:
   <<: *default
   compile: true
 
-  # Verifies that correct packages and versions are installed by inspecting package.json, yarn.lock, and node_modules
-  check_yarn_integrity: true
```

Comment:
Why is this a problem?

Reply:
This is enabled when we are building the image in CI, but when running in local is quite annoying https://github.com/rails/webpacker/issues/1135.